### PR TITLE
readd bob.ip.flandmark [skip appveyor]

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

**Comment by jakirkham** [comment]
I think this will be fixed once we have `setuptools` ( https://github.com/conda-forge/staged-recipes/pull/749 ). Thus far, I'm not encountering this import error with it.

**Comment by 183amir** [comment]
This is failing for good reasons! Setuptools is a runtime dependency of `bob` packages.

**Comment by jakirkham** [comment]
Was just about to ask that. Why is that the case?

**Comment by 183amir** [comment]
Because it uses `pkg_resources`!

**Comment by jakirkham** [comment]
Sure, I'm just wondering what it does with it.

**Comment by 183amir** [comment]
I remember in the early days, I had `setuptools` in runtime dependencies but you asked me to remove it. Since it was being pulled somehow, I ignored it and removed it.

**Comment by 183amir** [comment]
when I was doing the first `bob` package, `bob.extension`.

**Comment by 183amir** [comment]
It uses `pkg_resources` to handle 3 tier packages gracefully. All `__init__.py` files in bob packages use `pkg_resources` like this one: https://github.com/bioidiap/bob.extension/blob/master/bob/__init__.py

**Comment by jakirkham** [comment]
Agreed. That makes sense. Maybe we can do that first and come back here afterwards.

**Comment by 183amir** [comment]
ok all green now. Could you please merge this?

File: recipes/bob.ip.flandmark/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  md5: d1c9547362c8b7b819ff796a83de1f9c
+  url: https://pypi.io/packages/source/b/bob.ip.flandmark/bob.ip.flandmark-{{ version }}.zip
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
```
Comment (jakirkham) [inline recipes/bob.ip.flandmark/meta.yaml]: Could we please add the `toolchain`?

File: recipes/bob.ip.flandmark/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.9
+    export CFLAGS="${CFLAGS} -pthread"
+    export LDFLAGS="${LDFLAGS} -lpthread"
+fi
```
Comment (jakirkham) [inline recipes/bob.ip.flandmark/build.sh]: This should be replaced. Please see this PR ( https://github.com/conda-forge/thrift-cpp-feedstock/pull/4 ) for how to change this.

**Comment by 183amir** [comment]
@jakirkham adding `toolchain` here, breaks the builds on `pyhon 3.4`. Please take a look at the log of circleci. It builds fine for 2.7 but after that `-m -m` appears in the `CFLAGS` variable somehow.

**Comment by jakirkham** [comment]
Could you please try pinning `conda-build` to version 1.20.3 on Linux? It looks like there is some sort of bug that snuck in? The problem appears to be reproducible with `conda-build` 1.21.0 and appears to go away with 1.20.3. Not sure what the causes is yet though.

**Comment by jakirkham** [comment]
@msarahan, do you have any idea why this would happen?

**Comment by jakirkham** [comment]
@msarahan, is working on a `conda-build` fix. Also, Travis CI is pretty [backed up]( https://www.traviscistatus.com/incidents/nf3tg08pqg2d ). Could we please hold on this for a bit to give him time to work on a fix and Travis CI to settle down?

**Comment by 183amir** [comment]
Sure, but please don't cancel circle CI if possible.

**Comment by jakirkham** [comment]
No worries. Not canceling anything.

**Comment by 183amir** [comment]
@jakirkham this is finally building. Can we merge it now please?

**Pull request merged**